docs/sources/tempo/api_docs/_index.md

```diff
@@ -248,10 +249,11 @@ $ curl -G -s http://localhost:3200/api/search --data-urlencode 'tags=service.nam
 Ingester configuration `complete_block_timeout` affects how long tags are available for search.
 
 This endpoint retrieves all discovered tag names that can be used in search.  The endpoint is available in the query frontend service in
-a microservices deployment, or the Tempo endpoint in a monolithic mode deployment.
+a microservices deployment, or the Tempo endpoint in a monolithic mode deployment. The tags endpoint takes a scope that controls the kinds 
+of tags or attributes returned. If nothing is provided, the endpoint will return all resource and span tags.
 
 ```
-GET /api/search/tags
+GET /api/search/tags?scope=<resource|span|intrinsic>
```

Comment:
Is `scope` still present after the `v2` rev commits that were added?

Reply:
yup